modules/core/src/main/java/org/apache/ignite/internal/hlc/HybridTimestamp.java

```diff
@@ -239,4 +249,11 @@ public HybridTimestamp roundUpToPhysicalTick() {
             return new HybridTimestamp(getPhysical() + 1, 0);
         }
     }
+
+    /**
+     * Returns a byte array representing this timestamp.
+     */
```

Comment:
```suggestion
    /** Returns a byte array representing this timestamp. */
```

Reply:
On all methods of the class there are multi-line javadocs, so it doesn't make sense to use single-line javadoc here

Reply:
Ok.